dev-requirements.txt

```
@@ -5,7 +5,7 @@ azure-storage-blob>=12.5.0,<12.7.0
 azure-identity
 dask[dataframe]
 docker>=4.1.0,<5.0.0
-fsspec>=0.8.2,<=0.8.7
+fsspec>=0.9,<0.10.0
```

fsspec doesn't use semantic versioning, so the `<0.10` doesn't add any benefit.

```suggestion
fsspec>=0.9
```